### ISSUE TITLE
Chaos Rings 1 auto close

### ISSUE THREAD

**Issue by playerbano** [Vita3K/Vita3K-Android#202]
I tried opening the chaos rings 1 but it autoclose in loading. Loading in opening the game not the "in game" loading.

**Comment by metastigma** [comment]
> I tried opening the chaos rings 1 but it autoclose in loading. Loading in opening the game not the "in game" loading. Btw, Im using Redmi note 9 snapdragon 720g

same thing on windows 10 when vulcan renderer enabled (works fine with opengl)...r5 3600/16gb dd4 3200/1080ti

**Comment by playerbano** [comment]
> > I tried opening the chaos rings 1 but it autoclose in loading. Loading in opening the game not the "in game" loading. Btw, Im using Redmi note 9 snapdragon 720g
> 
> same thing on windows 10 when vulcan renderer enabled (works fine with opengl)...r5 3600/16gb dd4 3200/1080ti

How can I change this from vulkan to openGL? The drop down is disabled. 😅

**Comment by Raypuia** [comment]
> > 
> 
> maybe you better try android version of this game?...i'm using win only because phisical controls & also i'm translating it in realtime in hex editor to my language (playing on touchcreen is comfortable)

The Android version doesn't work on modern devices.
Trust me, after Android 5.0 the game stopped to work on most phones

**Comment by playerbano** [comment]
> > > 
> > 
> > 
> > maybe you better try android version of this game?...i'm using win only because phisical controls & also i'm translating it in realtime in hex editor to my language (playing on touchcreen is comfortable)
> 
> The Android version doesn't work on modern devices. Trust me, after Android 5.0 the game stopped to work on most phones

Yes thats right. I tried playing the android version of this on my device but it doesnt work. That's why Im trying to play it on emulator, but it doesnt work also.

**Comment by metastigma** [comment]
so...for now, only ios version works on jailed iphones & ipads (like mine ipad2) & win opengl
i tried to load custom libs in settings after choosing vulcan...but there are so many of them...
